src/components/Workspace.jsx

```diff
@@ -102,55 +99,39 @@ export default class Workspace extends React.Component {
     );
   }
 
-  _storeColumnRef(index, column) {
-    this.columnRefs[index] = column;
-  }
-
-  _storeDividerRef(divider) {
-    this.dividerRef = divider;
-  }
-
-  _handleDividerDrag(_, {deltaX, lastX, x}) {
-    const {onDragColumnDivider} = this.props;
-    onDragColumnDivider({
-      columnWidths: getNodeWidths(this.columnRefs),
-      dividerWidth: getNodeWidth(this.dividerRef),
-      deltaX,
-      lastX,
-      x,
-    });
-  }
-
   _renderEnvironment() {
```

Comment:
This change will make it substantially easier to decompose `<Workspace>` into several smaller presentational components, but that should come in a separate PR.